packages/Toast/src/Toast.styles.js

```diff
@@ -74,8 +74,8 @@ export const Content = styled.div`
 `;
 
 export const Toast = styled.div(
-  ({ isFixed, kind, zIndex, shouldRender }) => css`
-    align-items: flex-start;
```

Comment:
I think we need to retain `flex-start` or else we can end up with the icon and close button aligned in the middle for mult-line `<Toasts>`:

<img width="565" alt="Screen Shot 2021-10-29 at 2 01 10 PM" src="https://user-images.githubusercontent.com/14944896/139501239-e87da851-7efc-4824-ac81-d09542326fd5.png">